src/dotnet-install.ps1

```diff
@@ -619,6 +619,23 @@ function DownloadFile($Source, [string]$OutPath) {
     }
 }
 
+function SafeRemoveFile($Path) {
```

Comment:
nit: Should this function be named "Safe-Remove-File"? 
I am not sure I grasped the code style of the script, but most of the functions are named like that.

Reply:
According to https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/cmdlet-development-guidelines?view=powershell-7 none is correct in fact.  
I'm not really sure if we should fix naming convention in this PR. I think SafeRemoveFile is better choice to use as safe remove is not a verb,  alternative could be Remove-File without safe